<ui version="4.0" >
 <class>ArizonaProjectDlgUI</class>
 <widget class="QDialog" name="ArizonaProjectDlgUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Arizona Project Analysis</string>
  </property>
  <widget class="Q3GroupBox" name="rawDataProcessGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title" >
    <string>Load Raw Data</string>
   </property>
   <widget class="QPushButton" name="EllipsoidFileButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>55</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select Ellipsoid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectRawFileButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select Raw File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Load data</string>
    </property>
   </widget>
   <widget class="QLabel" name="rawFileSelectedTextLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>20</y>
      <width>290</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>No file selected...</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ellipsoidFileSelectedTextLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>55</y>
      <width>290</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>No file selected...</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="testGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>240</y>
     <width>401</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title" >
    <string>Grasp Analysis</string>
   </property>
   <widget class="QPushButton" name="loadObjectFileButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Load Object...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevExpButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextExpButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>59</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>></string>
    </property>
   </widget>
   <widget class="QLabel" name="expNumberLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>66</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>0/0</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="showGWSButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>208</x>
      <y>72</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show GWS</string>
    </property>
   </widget>
   <widget class="Q3GroupBox" name="testResultGroup" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>20</y>
      <width>111</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title" >
     <string>Current Experiment</string>
    </property>
    <widget class="QLabel" name="typeLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>13</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Type:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="CMLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>46</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>CM:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="qualityLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>76</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Quality:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="forceLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>110</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Min force:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="Q3ButtonGroup" name="buttonGroup1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>270</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title" >
     <string>Test</string>
    </property>
    <property name="exclusive" >
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="radioQual" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Quality</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioForce" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Force</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioContact" >
     <property name="geometry" >
      <rect>
       <x>122</x>
       <y>20</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testAllButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>190</x>
       <y>30</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Test all...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fnTextBox" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>40</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel1" >
     <property name="geometry" >
      <rect>
       <x>14</x>
       <y>36</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>File Name</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="objectFileSelectedTextLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>20</y>
      <width>170</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>No file selected...</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="isBuildIn3DCheckBox" >
    <property name="geometry" >
     <rect>
      <x>132</x>
      <y>66</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>directly3D</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isFlippedCheckBox" >
    <property name="geometry" >
     <rect>
      <x>132</x>
      <y>86</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>flip normal</string>
    </property>
   </widget>
  </widget>
  <widget class="Q3GroupBox" name="groupBox3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>144</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Save as VGR</string>
   </property>
   <widget class="QPushButton" name="destinationFolderButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select Folder...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveVgrButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>54</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="destinationFolderLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>20</y>
      <width>290</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>No folder selected...</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
